### PR TITLE
Documents how to disable or replace the default cluster health check

### PR DESCRIPTION
**Pull request by ignasi35**
As mentioned in https://github.com/akka/akka-management/issues/470#issuecomment-465004498

### CODE REVIEW

**Comment by ignasi35** [comment]
the build fails for different reasons:

 * fail to bind a port (flaky test?)
 * can't access a key file (some change on travis infra?)

File: cluster-http/src/main/resources/reference.conf [hunk 1]
```diff
@@ -26,6 +26,8 @@ akka.management {
 akka.management {
   health-checks {
     readiness-checks {
+      # Default HHealth chheck for cluster. Overwri the setting to replace it with
```
Comment (patriknw) [inline cluster-http/src/main/resources/reference.conf]: typo HH and Overwri

**Comment by patriknw** [comment]
I fixed something that I think could have caused the port bind problems.

There is another build failure in the integration tests that I don't know what it is about:
```
Error in configuration: unable to read client-key /home/travis/.minikube/client.key for minikube due to open /home/travis/.minikube/client.key: permission denied
```

File: cluster-http/src/main/resources/reference.conf (hunk 1 above)
Comment (ignasi35) [reply cluster-http/src/main/resources/reference.conf]:
```suggestion
      # Default health check for cluster. Overwrite the setting to replace it with
```

My `h` key is acting up (see `chheck` too).

**Pull request merged**